### DISCUSSION TITLE
Optimizing Antenna Performance in an Indoor Propagation Environment (Project 206)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#37] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 206](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Optimizing%20Antenna%20Performance%20in%20an%20Indoor%20Propagation%20Environment)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by arunvishal2712** [comment]
Hi, I am a student at St. Joseph's College of Engineering, OMR, Chennai, and interested in working and collaborating on this project.

**Comment by wonka360** [comment]
Hello please I am a fellow participant in this project and I have been working on it for over 3 weeks now. Please to whoever is in charge, I have 2 major questions to ask. The first part of this project needs that we optimize the signal strength in an indoor environment. An obvious way to do that will be to maximize the gain of the antenna element, but since we are dealing with wireless indoor propagation my question is that is it also required of us to optimize the orientation of the transmitting antenna.
    The second question is this. The second part requires that we train a machine learning based prediction model based on the simulated data in the first part. My question is, after we have optimized the transmitting antenna and done some ray tracing using RF propagation tool box are we supposed to use this data to train the model, and if so what is the expected result of the machine learning model. Like what inputs do you expect the model to receive and what outputs/predictions do you expect. Please I need clarifications on the inputs and outputs. Thank you.
